### PR TITLE
First issue

### PR DESCRIPTION
**Pull request by redmouce**

### CODE REVIEW

**Comment by redmouce** [comment]
Need to be improved with watchers. They'r nearly ready, but I'am falling asleep, so they'll come soon

**Comment by redmouce** [comment]
OFC I don't want to merge into Master. Waiting for own branch

**Comment by techno-aads** [comment]
```
Starting audit...
[ERROR] /Users/mikhail.nechaev/Programming/java-advanced-2018-autumn/src/ru/ifmo/task1/Logger.java:8:8: Unused import - java.sql.SQLOutput. [UnusedImports]
[ERROR] /Users/mikhail.nechaev/Programming/java-advanced-2018-autumn/src/ru/ifmo/task1/Watcher.java:5: Using the '.*' form of import should be avoided - java.nio.file.*. [AvoidStarImport]
[ERROR] /Users/mikhail.nechaev/Programming/java-advanced-2018-autumn/src/ru/ifmo/task1/Watcher.java:8:8: Unused import - java.util.LinkedHashSet. [UnusedImports]
Audit done.
Checkstyle ends with 3 errors.
```

File: src/ru/ifmo/task1/Walker.java [hunk 1]
```diff
@@ -11,20 +26,129 @@
...
+        }
+    }
+
+    private ParseArgsResult parseArgs(String... args) throws IOException {
+        if (args.length < 2) {
+            logger.error("not enough parameters have been passed");
+            return new ParseArgsResult(null, null, false);
+        }
+
+        File first = new File(args[0]);
```
Comment (techno-aads) [inline src/ru/ifmo/task1/Walker.java]: Path = Paths.get

**Pull request closed without merging**